README.md

```diff
@@ -224,6 +224,22 @@ field(:status).options(["draft", "published"])
 field(:status).policy(:options, ["draft", "published"])
 ```
 
+### :length
+
+Specify value's length constraints
+ - `min:` - The attribute cannot have less than the specified length.
+ - `max`  - The attribute cannot have more than the specified length.
+ - `eq`   - The attribute should be exactly than the specified length.
```

Comment:
did you mean: exactly equal to the specified length?

Reply:
yep )